### DISCUSSION TITLE
Github Education failing verification

### DISCUSSION THREAD

**Discussion by marchundleydegree** [community/community#146385] [GitHub Education]
### Hi everyone,

Hi,

I am enrolled on an apprenticeship provided via the apprenticeship levy through my employer. The apprenticeship is being provided by QA Limited and is accredited by the University of Roehampton.

This is provided via distance learning/videoconferencing.

I have attempted to upload evidence of my enrolment, however I am continually being rejected.

I have received the following reasons :

_1. Please ensure that your academic affiliation document contains your last name exactly as it appears in your [GitHub billing information](https://github.com/settings/billing/payment_information). If you have only a single legal name, enter it in both the first and last name fields of your GitHub billing information._

I have corrected this in the billing section, the letter of confirmation matches both my names and the address.

_2. The image you selected does not appear to contain your school name. Your complete school name must appear in your document, not only the school logo. You may include multiple documents together. If your official document is not in English then you may photograph the original next to an English translation._

This is difficult as the apprenticeship is provided by QA Higher learning (QA Limited) and is affiliated with the University of Roehampton. I do not have a University ID Card and the letter of confirmation only has the QA company logo.

_3. Your document does not appear to include a date demonstrating current academic affiliation. For countries utilizing non-standard calendars, you may need to capture the original document beside one with a converted date. You may include multiple documents in your image, so long as they are legible._

This appears to be an issue with the character recognition from the image as the letter clearly states both start and end dates within the required time ranges.

_4. Please ensure that your academic affiliation document contains your first name exactly as it appears in your [GitHub billing information](https://github.com/settings/billing/payment_information). Variations of your name and nicknames are unacceptable._

This is accurate based on the letter and my Github account details

_5. You are unlikely to be verified until you have completed your [GitHub user profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/personalizing-your-profile) with your full name exactly as it appears in your academic affiliation document. Please do not use a variation of your name or a nickname. Once you have updated your profile information log out and log back into GitHub before re-applying._

These have been completed and I have logged out and back in again.

Any advice will be gratefully received, along with any possibility of contacting a human a Github who can assist with this issue.

Thanks in advance

**Comment by Stansbury-C** [comment]
Hiya. 
I was completing my Level 7 with Cambridge Spark, an organisation until recently qualified me for the student developer pack. I was then, a few months after asked to verify my status and I was told that Cambridge Spark no longer qualifies. 
I'm not sure if the programme has changed of just the wording as it now states a Degree or Diploma issuing faculty. 

Be at least 13 years old.
Be currently enrolled in a degree or diploma granting course of study from a recognized educational institution.
Be able to provide documentation from your school which demonstrates your current student status.

I tried everything, submitting my agreements, my Cambridge edu email etc and nothing. The Apprenticeship does not seem to fall under the program. 

Do you have a roehampton.ac.uk address? Someone in Student Services may be able to provide you with one as technically you are under their faculty even if through a provider. Worth a try?

**Reply by marchundleydegree** [reply]
Thanks Stansbury-C,

I guess I may be in a similar situation as you were.

I used a submitted doc from QA (the educational institute) , we were not issued a Roehampton email address or student ID's despite being as you say under their remit.

I will however try your suggestion of contacting their Student Services, got to be worth a go.

Thank you
